invenio_app_ils/mail/tasks.py

```diff
@@ -69,4 +121,7 @@ def log_error_mail(request, exc, traceback, data, **kwargs):
         exception=repr(exc),
         data=data,
     )
+
+    create_email_log_entry(data, "Error" + exc)
```

Comment:
It would be nice to test this: I think it is missing the space between `Error` and `exc` and we are using the function `repr` above, might be worth to have a look how it looks like.